client/db/bolt/db.go

```diff
@@ -299,6 +300,65 @@ func (db *BoltDB) CreateAccount(ai *dexdb.AccountInfo) error {
 	})
 }
 
+// DeleteAccount removes the account by host.
+func (db *BoltDB) DeleteAccount(ai *dexdb.AccountInfo) error {
```

Comment:
This method is only used in this package, so I don't think it needs to be exported (capitalized), unless this is satisfying an interface somewhere.

Reply:
Fixed for next push.